evennia/contrib/tutorials/evadventure/chargen.py

```diff
@@ -321,9 +334,10 @@ def start_chargen(caller, session=None):
         "node_chargen": node_chargen,
         "node_change_name": node_change_name,
         "node_swap_abilities": node_swap_abilities,
+        "node_apply_character": node_apply_character,
     }
 
     # this generates all random components of the character
     tmp_character = TemporaryCharacterSheet()
-
-    EvMenu(caller, menutree, startnode="node_chargen", session=session, tmp_character=tmp_character)
+    
+    EvMenu(caller, menutree, startnode="node_chargen", session=session, startnode_input=('sgsg', {"tmp_character":tmp_character}))
```

Comment:
I suppose anything could go in the first string of the tuple here, but it's confusing to put '`sgsg'`, it would be better to just pass the empty string.